client/my-sites/promote-post-i2/style.scss

```diff
@@ -256,7 +265,7 @@ body.is-section-promote-post-i2 {
 			margin-right: 0;
 
 			@media screen and (max-width: $break-medium) {
-				margin: 0;
+				margin: 0 0 0 auto; // Align Credits tab to the right
```

Comment:
I checked in the designs, and the mobile version of the tabs shows the credits next to the other two. 
Maybe we can start a conversation with Bart regarding this? I am not sure what alignment is correct here.